SimTracker/TrackTriggerAssociation/python/StubAssociator_cfi.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+StubAssociator_params = cms.PSet (
+  InputTagTTStubDetSetVec = cms.InputTag( "TTStubsFromPhase2TrackerDigis",     "StubAccepted"    ), #
+  InputTagTTClusterAssMap = cms.InputTag( "TTClusterAssociatorFromPixelDigis", "ClusterAccepted" ), #
+  InputTagTTStubAssMap    = cms.InputTag( "TTStubAssociatorFromPixelDigis",    "StubAccepted"    ), #
+  BranchReconstructable   = cms.string  ( "Reconstructable" ),                                      #
```

Comment:
Add comment explaining that BranchReconstrucable & BranchSelection are the names of the two output StubAssociation collections, made with reconstruction and "used for tracking efficiency" TrackingParticles.